Mathlib/RepresentationTheory/Homological/GroupHomology/LowDegree.lean

```diff
@@ -0,0 +1,268 @@
+/-
+Copyright (c) 2025 Amelia Livingston. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Amelia Livingston
+-/
+import Mathlib.RepresentationTheory.Homological.GroupHomology.Basic
+
+/-!
+# The low-degree homology of a `k`-linear `G`-representation
+
+Let `k` be a commutative ring and `G` a group. This file will contain specialised API for
+the cycles and group homology  of a `k`-linear `G`-representation `A` in degrees 0, 1 and 2.
+In `RepresentationTheory/Homological/GroupHomology/Basic.lean`, we define the `n`th group homology
+of `A` to be the homology of a complex `inhomogeneousChains A`, whose objects are
+`(Fin n →₀ G) → A`; this is unnecessarily unwieldy in low degree. Here, meanwhile, we will define
+the one and two cycles and boundaries as submodules of `G →₀ A` and `G × G →₀ A`, and provide maps
+to `H1` and `H2`.
+
+-/
+
+universe v u
+
+noncomputable section
+
+open CategoryTheory Limits Representation Rep Finsupp
+
+variable {k G : Type u} [CommRing k] [Group G] (A : Rep k G)
+
+namespace groupHomology
+
+section Chains
+variable [DecidableEq G]
+
+/-- The 0th object in the complex of inhomogeneous chains of `A : Rep k G` is isomorphic
+to `A` as a `k`-module. -/
+def zeroChainsIso : (inhomogeneousChains A).X 0 ≅ A.V :=
+  (LinearEquiv.finsuppUnique _ _ _).toModuleIso
+
+/-- The 1st object in the complex of inhomogeneous chains of `A : Rep k G` is isomorphic
+to `G →₀ A` as a `k`-module. -/
+def oneChainsIso : (inhomogeneousChains A).X 1 ≅ ModuleCat.of k (G →₀ A) :=
+  (Finsupp.domLCongr (Equiv.funUnique (Fin 1) G)).toModuleIso
+
+/-- The 2nd object in the complex of inhomogeneous chains of `A : Rep k G` is isomorphic
+to `G² →₀ A` as a `k`-module. -/
+def twoChainsIso : (inhomogeneousChains A).X 2 ≅ ModuleCat.of k (G × G →₀ A) :=
+  (Finsupp.domLCongr (piFinTwoEquiv fun _ => G)).toModuleIso
+
+/-- The 3rd object in the complex of inhomogeneous chains of `A : Rep k G` is isomorphic
+to `G³ → A` as a `k`-module. -/
+def threeChainsIso : (inhomogeneousChains A).X 3 ≅ ModuleCat.of k (G × G × G →₀ A) :=
+  (Finsupp.domLCongr ((Fin.consEquiv _).symm.trans
+    ((Equiv.refl G).prodCongr (piFinTwoEquiv fun _ => G)))).toModuleIso
+
+end Chains
+
+section Differentials
+
+/-- The 0th differential in the complex of inhomogeneous chains of `A : Rep k G`, as a
+`k`-linear map `(G →₀ A) → A`. It sends `single g a ↦ ρ_A(g⁻¹)(a) - a.` -/
+def dZero : ModuleCat.of k (G →₀ A) ⟶ A.V :=
```

Comment:
What do you think of calling this `d₁₀`? That way it would be visually very clear that it should fit in a square with `.d 1 0`.

Reply:
Thank you for these suggestions and sorry for the delay - I would want to bring group cohomology in line with these changes, and the PR would be about [600 lines long](https://github.com/leanprover-community/mathlib4/compare/master...101damnations:mathlib4:gpcohlowdegrenaming) - can I get away with that if it's just renaming?

Reply:
Just to be clear -- the notational changes have now been made in this PR.